### ISSUE TITLE
"Error Creating Machine" "code E_FAIL (0x80004005)" "--type headless failed:"

### ISSUE THREAD

**Issue by ArthurBohn** [ladybug-tools/butterfly#333]
Hi everyone,

Trying to install as your tutorial on youtube, I always have the same problem ( it installed/uninstalled at least 10 times..).

I have a Windows 10 Home.
During the installation, in the first window terminal, I have the message as sent below. ( I copied it in .txt )
I first thought it was my Avast antivirus, so I disabled it.
Then I tried to install checking the NDIS5 driver instead of NDIS6. 
I tried to install VirtualBox 5.0.12 on my own, by the command prompt using "-msiparams NETWORKTYPE=NDIS5" .
Same error...

Thank you very much for your time and your explanations.
Arthur
[bug_install_openfoam.txt](https://github.com/ladybug-tools/butterfly/files/1787303/bug_install_openfoam.txt)

**Comment by albesaca** [comment]
you're not alone

https://github.com/ladybug-tools/butterfly/issues/331

**Comment by ArthurBohn** [comment]
Hello albesaca, actually I installed Oracle Virtual Box ( the newest ) and it runs for me now !

Sorry if I maybe doubled the topic.. I thought maybe windows Home was the problem..
Actually I installed the newest version of VirtualBox 5.2.8 that I found on the Oracle website. 
For this, I uninstalled the whole package ( openFoam, VBox and docker ) , then I installed the newest version of VBox. After that I installed openFoam without checking VirtualBox in the list. 
With that it worked like the tutorial.Everything works, also the butterfly plugin ! 
Happy ;)
Thx a lot for all the work behind what we see..
